docs/index.bs

```diff
@@ -3175,6 +3235,78 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. Return |response|.
   </section>
 
+  <section algorithm>
+    <h3 id="verify-router-rule-algorithm"><dfn>VerifyRouterRule</dfn></h3>
+
+      : Input
+      :: |rule|, a {{RouterRule}}
+      :: |serviceWorker|, a [=/service worker=]
+      : Output
+      :: a boolean
+
+      1. If |rule|["{{RouterRule/condition}}"]["{{RouterCondition/urlPattern}}"] is the empty string, return false.
+      1. Let |rawPattern| be |rule|["{{RouterRule/condition}}"]["{{RouterCondition/urlPattern}}"].
+      1. If |rawPattern| is a [=string=], then:
+        1. Let |baseURL| be |serviceWorker|'s [=service worker/script url=].
```

Comment:
Just to clarify, `[=service worker/script url=]` is a main script URL, correct?

Reply:
I understand so.
It refers https://w3c.github.io/ServiceWorker/#dfn-script-url